tests/test_cli.py

```diff
@@ -314,3 +316,48 @@ def test_vertex_run(tmp_path_factory):
             service_account=None,
             network=None,
         )
+
+
+@patch("docker.api.client.APIClient.push")
+@patch("docker.api.client.APIClient.build")
+def test_component_build(mock_build, mock_push):
+    """Test that the build command works as expected."""
+    args = argparse.Namespace(
+        component_dir=Path("example_component"),
```

Comment:
I think you need to update this to  `Path(__file__).parent / "example_component"` for the test to work, it's currently failing